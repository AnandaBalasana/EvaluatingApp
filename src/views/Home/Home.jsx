import React from 'react'

class Home extends React.Component {
  render() {
    return (
      <div className="home">
        <p>This is home page. Choose section in menu.</p>
      </div>
    )
  }
}

export default Home
