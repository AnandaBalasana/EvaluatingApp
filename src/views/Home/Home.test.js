import React from 'react'
import { render } from '@testing-library/react'
import Home from './Home'

test('renders home component', () => {
    const { getByText } = render(<Home />)
    const textElement = getByText(/home page/i)
    expect(textElement).toBeInTheDocument()
})

test('snapshot', () => {
    const comp = render(<Home />)
    expect(comp).toMatchSnapshot()
})
