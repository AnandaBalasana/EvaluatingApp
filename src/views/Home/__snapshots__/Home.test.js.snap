// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div>
  <div
    class="home"
  >
    <p>
      This is home page. Choose section in menu.
    </p>
  </div>
</div>
`;
